imagetest/fixtures.go

```diff
@@ -156,3 +156,51 @@ func (t *TestVM) EnableSecureBoot() {
 		}
 	}
 }
+
+// EnableNetwork ensure the create network step run before create test VMs and
+// current test VMs in workflow use it
+func (t *TestVM) EnableNetwork(networkName, subnetworkName, aliasIPRange, rangeName string) error {
+	firstStep, ok := t.testWorkflow.wf.Steps[createDisksStepName]
+	if !ok {
+		return fmt.Errorf("failed resolve first step")
+	}
+
+	createNetworkStepName, ok := t.testWorkflow.wf.Steps["create-network-"+t.name]
+	if !ok {
+		return fmt.Errorf("create-network-%s step missing", t.name)
+	}
+	createSubNetworkStepName, ok := t.testWorkflow.wf.Steps["create-sub-network-"+t.name]
+	if !ok {
+		return fmt.Errorf("create-sub-network-%s step missing", t.name)
+	}
+	if err := t.testWorkflow.wf.AddDependency(createSubNetworkStepName, createNetworkStepName); err != nil {
+		return err
+	}
+	if err := t.testWorkflow.wf.AddDependency(firstStep, createSubNetworkStepName); err != nil {
+		return err
+	}
+
+	for _, i := range t.testWorkflow.wf.Steps[createVMsStepName].CreateInstances.Instances {
+		if i.Name == t.name {
+			network := compute.NetworkInterface{
+				Network:    networkName,
+				Subnetwork: subnetworkName,
+				AccessConfigs: []*compute.AccessConfig{
+					{
+						Type: "ONE_TO_ONE_NAT",
+					},
+				},
+				AliasIpRanges: []*compute.AliasIpRange{
+					{
+						IpCidrRange:         aliasIPRange,
+						SubnetworkRangeName: rangeName,
+					},
+				},
+			}
+
+			i.NetworkInterfaces = append(i.NetworkInterfaces, &network)
```

Comment:
do we intend to support multiple network interfaces? what happens if someone calls EnableNetwork 2 times?